bin/migration/tables/portalaccess.py

```diff
@@ -75,6 +76,8 @@ def migrate_data(self, destination_cursor, source_user_data):
                     status = "INVITATION_SENT"
 
                 invited_at, registered_at = dataverse_info.get(user_id, (None, None))
+                invited_at = datetime.now() if invited_at is None else invited_at
```

Comment:
The invited_at var could be initialised at line 77, then checked if it needs to changed !

Also, what about the value for registered_at, do we want to keep its value of None ?